app/src/main/java/com/android/streetworkapp/model/moderation/PerspectiveAPIRepository.kt

```diff
@@ -56,56 +58,55 @@ class PerspectiveAPIRepository(private val client: OkHttpClient) : TextModeratio
    * @return TextEvaluationResult.Success if the API could process the content,
    *   TextEvaluationResult.Error if an error was encountered
    */
-  private fun getTextAnnotations(content: String): PerspectiveAPIEvaluationResult {
-    // I don't perform any checks on the content, I'll let the api handle the error checking
+  private suspend fun getTextAnnotations(content: String): PerspectiveAPIEvaluationResult {
+    return withContext(Dispatchers.IO) {
```

Comment:
Suggestion: testing might be hindered by using a hardcoded dispatcher, i would recommend using `Dispatchers.Default` instead

Reply:
Why would the testing being hindered? My reasoning for the choice of `Dispatchers.IO` is that the operation we're doing is more network heavy than anything else. The api call should be the thing that takes the most amount of time here.

Reply:
This is something that was noted by sonarcloud mainly, injectable dispatchers ease testing by allowing tests to inject more deterministic dispatchers. But i guess this is a problem that we can deal with when we actually encounter it, you can do as you want for now